### DISCUSSION TITLE
Contributor Call: 1 April 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3686] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: April Fools' Day (1 April) 2023 ⚠**4pm**⚠ UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/apr-1-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.
⚠Watch out for timezone changes!

### Agenda
- TBD

**Comment by BenJamesBen** [comment]
Should the structure of the Guides section of dev documentation be changed to have a top-level Writers Guide section? The Markdown Guide and Documentation Structure docs would be moved from the top-level down into the Writers Guide section.

Proposed: https://freesewing-dev-website-git-documentation-gui-e5dff6-benjamesben.vercel.app/guides/writers
Current: https://freesewing-dev-website-git-working-tortuga-6-benjamesben.vercel.app/guides

Alternatives include leaving the Markdown Guide at the top-level.

**Comment by tangerineshark** [comment, marked as answer]
**Contributor Call Notes**
(by @tangerineshark , so if it doesn't make sense, that's why)

- Newsletter now goes out faster via SMTP after Joost got a new phone from some faith-in-humanity-restoring tech shop folks
- Newsletter also mentions this, but FS is changing the subscription model to pay-as-you-can with recurring and one-time options, and moving to Stripe
- @BenJamesBen is working on the docs mentions in his comment and everyone loves it
- @eriese got a haircut and it looks awesome
- @starfetch is building a sewing space and it involves moving a jealousy-inducing number of books
- @SeaZeeZee finished a modified set of Scroop Patterns stays, and they're great

Updates from @joostdecock 
_Splitting hooks:_
- an explanation of hooks for Karen: hooks are components like a header, language picker, etc., but they don't have a visual component (e.g., use backend, use account, etc.))
- hooks used to be all together, and you had to use them all, but now you can pick and choose, thus they are split... ta da! Splitting hooks!

_Login flow updates:_
- (is this also related to hooks? Karen has them written in the same section) Dev environment will be able to log into FS account and use data stored there (but this won't merge for a while b/c you'd be able to log in without it working  (so it just lives in @joostdecock 's branch)
- Email address only in V3, no password required. Instead, you just get a login link, but you can still set a username and password if you want (plus 2FA 🎉 )

_Changing navigation:_
- You need the breadcrumbs (those lil guys who tell you where on the website you are) to be in your language when you use FS, which is currently handled with one massive "navigation blob" -> too much data.
- Now, mini breadcrumbs are in the navigation bar, but not the entire tree, which uses less data, looks pretty much the same to the user, but changes a lot of things under the hood

Translation:
- Now linking translations to the stuff they translate (e.g., pattern options), so if we get rid of something, we can also drop the translation strings, instead of just accumulating a massive compendium of strings we're just dragging around like a big bag of all the threads you've ever snipped for your entire life

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
